chpl/chpl-service/src/main/java/gov/healthit/chpl/search/SearchRequestNormalizer.java

```
@@ -259,6 +261,33 @@ private void normalizeSvapOperator(SearchRequest request) {
         }
     }
 
+    private void normalizeRiskManagementSummaryInformationOptions(SearchRequest request) {
+        if (!CollectionUtils.isEmpty(request.getRiskManagementSummaryInformationOptionsStrings())
+                && CollectionUtils.isEmpty(request.getRiskManagementSummaryInformationOptions())) {
+            try {
+                request.setRiskManagementSummaryInformationOptions(
+                        request.getRiskManagementSummaryInformationOptionsStrings().stream()
+                                .filter(option -> !StringUtils.isBlank(option))
+                                .map(option -> convertToRiskManagementSummaryInformationSearchOption(option))
+                                .filter(option -> option != null)
+                                .collect(Collectors.toSet()));
+            } catch (Exception ignore) {
+            }
+        }
+    }
+
+    private RiskManagementSummaryInformationlSearchOptions convertToRiskManagementSummaryInformationSearchOption(String option) {
+        if (StringUtils.isBlank(option)) {
+            return null;
+        }
+        RiskManagementSummaryInformationlSearchOptions convertedOption = null;
+        try {
+            convertedOption = RiskManagementSummaryInformationlSearchOptions.valueOf(option.toUpperCase().trim());
+        } catch (Exception ex) {
+        }
```

Why are we catching and swallowing an exception?